ntbs-service/wwwroot/source/Components/AutocompleteSelect.ts

```diff
@@ -15,7 +15,12 @@ const AutocompleteSelect = Vue.extend({
         };
         if (this.$props.validate) {
             options.onConfirm = (val: any) => {
-                this.onConfirm(val);
+                this.onConfirmValidate(val);
+            }
+        };
+        if (this.$props.filter) {
+            options.onConfirm = (val: any) => {

```

Comment:
I think it's confusing having separate `onConfirm` methods - I think it would be better to bring the `if` statement inside a single `onConfirm` method.